IPP linker workaround for Intel compiler on Linux
Reported: https://github.com/opencv/opencv/pull/19681
Fixed: https://github.com/opencv/opencv/commit/6465e393b697cfdba1d38c155e245598ab302f33
Likely not needed for OpenCV version > 4.5.2
An alternative is to use -DOPENCV_FORCE_IPP_EXCLUDE_LIBS=ON 
--- opencv-4.5.1/cmake/OpenCVFindLibsPerf.cmake
+++ opencv-4.5.1/cmake/OpenCVFindLibsPerf.cmake
@@ -29,7 +29,7 @@ if(WITH_IPP)
     if(OPENCV_FORCE_IPP_EXCLUDE_LIBS
         OR (HAVE_IPP_ICV
             AND UNIX AND NOT ANDROID AND NOT APPLE
-            AND (CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+            AND CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang|Intel"
         )
         AND NOT OPENCV_SKIP_IPP_EXCLUDE_LIBS
     )
